main.nf

```diff
@@ -30,28 +30,32 @@ def helpMessage() {
      --genome [str]                     Name of iGenomes reference
      --single_end [bool]                Specifies that the input is single end reads
      --comprehensive [bool]             Output information for all cytosine contexts
-     --cytosine_report [bool]           Output stranded cytosine report during Bismark's bismark_methylation_extractor step.
      --ignore_flags [bool]              Run MethylDackel with the flag to ignore SAM flags.
      --meth_cutoff [int]                Specify a minimum read coverage to report a methylation call during Bismark's bismark_methylation_extractor step.
-     --min_depth [int]                  Specify a minimum read coverage for MethylDackel to report a methylation call.
+     --min_depth [int]                  Specify a minimum read coverage for MethylDackel to report a methylation call or for biscuit pileup.
      --methyl_kit [bool]                Run MethylDackel with the --methyl_kit flag to produce files suitable for use with the methylKit R package.
      --skip_deduplication [bool]        Skip deduplication step after alignment. This is turned on automatically if --rrbs is specified
      --non_directional [bool]           Run alignment against all four possible strands
      --save_align_intermeds [bool]      Save aligned intermediates to results directory
      --save_trimmed [bool]              Save trimmed reads to results directory
+	 --save_pileup_file [bool]          Save vcf-pileup and index-vcf files from biscuit aligner to results directory
+	 --save_snp_file					Save SNP bed-file from biscuit to results directory. Relevant only if '--epiread' is specified
      --unmapped [bool]                  Save unmapped reads to fastq files
      --relax_mismatches [bool]          Turn on to relax stringency for alignment (set allowed penalty with --num_mismatches)
      --num_mismatches [float]           0.6 will allow a penalty of bp * -0.6 - for 100bp reads (bismark default is 0.2)
      --known_splices [file]             Supply a .gtf file containing known splice sites (bismark_hisat only)
      --slamseq [bool]                   Run bismark in SLAM-seq mode
      --local_alignment [bool]           Allow soft-clipping of reads (potentially useful for single-cell experiments)
-     --bismark_align_cpu_per_multicore [int] Specify how many CPUs are required per --multicore for bismark align (default = 3)
-     --bismark_align_mem_per_multicore [str] Specify how much memory is required per --multicore for bismark align (default = 13.GB)
+	 --soloWCGW_file [path]             soloWCGW file, to intersect with methyl_extract bed file. soloWCGW for hg38 can be downlaod from: www.cse.huji.ac.il/~ekushele/solo_WCGW_cpg_hg38.bed. EXPERMINTAL!
```

Comment:
How was this file generated?

Reply:
I removed this step, because it's a bit complicated to explain how they were generated... Done